### ISSUE TITLE
"Load More" Button Issue

### ISSUE THREAD

**Issue by TheSoloJake** [toggl-open-source/toggldesktop#2230]
Case is experiencing an issue where clicking on the Load More button in the Windows app only generates entries from the previous nine days. The button disappears after that.

Case 519139

**Comment by IndrekV** [comment]
`Load more` button can be clicked only once per session. If clicked it should fetch entries from the last 60 days and then button disappears.

**Comment by jayvdb** [comment]
I have data only back to Oct 2.  When I click Load More on Linux, it only shows back as far as Oct 3, and only shows about half of Oct 3.

**Comment by skel35** [comment]
Closing as a duplicate of #3270
